### ISSUE TITLE
Values being swapped in 1.2.1

### ISSUE THREAD

**Issue by jljr222** [scssphp/scssphp#181]
Hi All,

I am using scssphp in my project (v1.2.1) and I have noticed that when I compile my CSS, it's swapping some important values. Here is an example:

**Compiled by scssphp**
```
.alert:before,#twofaactivation .errorbox:before {
    position: absolute;
    left: 50%;
    transform: translate(-50%, 0);						 
    font-size: 1.25rem
}
```

**Compiled by prepros app**
```
#twofaactivation .errorbox:before,.alert:before {
    position: absolute;
    top: 50%;
    -webkit-transform: translate(0,-50%);
    transform: translate(0,-50%);
    font-size: 1.25rem
}
```

Those values are being generated by the following mixin in my SCSS:
```
@mixin centerer($horizontal: true, $vertical: true) {
	position: absolute;
	@if ($horizontal and $vertical) {
		top: 50%;
		left: 50%;
		transform: translate(-50%, -50%);
	} @else if ($horizontal) {
		left: 50%;
		transform: translate(-50%, 0);
	} @else if ($vertical) {
		top: 50%;
		transform: translate(0, -50%);
	}
}
```

The above mixin is being called as such in my SCSS:
`@include centerer($horizontal: false);
`

Should I roll back to an older version or is there a newer (dev) version I can set in composer to use?

**Comment by Cerdic** [comment]
I can reproduce with the following test-case: 
```
@mixin centerer($horizontal: true, $vertical: true) {
	position: absolute;
	@if ($horizontal and $vertical) {
		top: 50%;
		left: 50%;
		transform: translate(-50%, -50%);
	} @else if ($horizontal) {
		left: 50%;
		transform: translate(-50%, 0);
	} @else if ($vertical) {
		top: 50%;
		transform: translate(0, -50%);
	}
}

#twofaactivation .errorbox,.alert {
    &:before {
        @include centerer($horizontal: false);     
    }
}
```

gives with SassMeister
```
#twofaactivation .errorbox:before, .alert:before {
  position: absolute;
  top: 50%;
  transform: translate(0, -50%);
}
```

whereas scssphp returns

```
#twofaactivation .errorbox:before, .alert:before {
  position: absolute;
  left: 50%;
  transform: translate(-50%, 0); }
```

`left` vs `top` in the result output, meaning there is an issue with arguments values

**Comment by jljr222** [comment]
I was able to get around this by slightly modifying the mixin and the way it's called:

Updated mixin:
```
@mixin centerer($horizontal: true, $vertical: true) {
	position: absolute;
	@if $horizontal and $vertical {
		top: 50%;
		left: 50%;
		transform: translate(-50%, -50%);
	} @else if $horizontal {
		left: 50%;
		transform: translate(-50%, 0);
	} @else if $vertical {
		top: 50%;
		transform: translate(0, -50%);
	}
}
```

Updated include:
`@include centerer(false, true);`

Not sure if that helps, but at least I can work around it.

**Comment by Cerdic** [comment]
the faulty commit seems to be f9f864318f08c8da7b7fbb5f796ab8ec3dd9f04a

**Comment by Cerdic** [comment]
ok problem solved, I did not fully fix things around `@if` in fixing #165
